build/Dockerfile

```diff
@@ -32,4 +32,10 @@ ADD https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-serv
 
 ADD https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-server/main/tests/test_models/AbsComponentModelWeight/Full/KerasCompWeightFullPipeline/KerasCompWeightFullPipeline.json /var/lib/kepler/data/KerasCompWeightFullPipeline.json
 
+# pre install kernel sources
+RUN mkdir -p /usr/share/kepler/kernel_sources
+
+COPY --from=quay.io/sustainable_computing_io/kepler_kernel_source_images:ubi8 /usr/src/kernels /usr/share/kepler/kernel_sources
```

Comment:
will this be kept in the released image? how big it is for the copied files?

Reply:
each kernel-devel is about 5M